src/main/java/org/mockito/junit/MockitoJUnit.java

```diff
@@ -33,6 +35,19 @@ public static MockitoRule rule() {
         return new JUnitRule(Plugins.getMockitoLogger(), Strictness.WARN);
     }
 
+    /**
+     * Creates a rule instance that initiates &#064;Mocks and is a {@link TestRule}. Use this method
```

Comment:
I suggest to make the recommended use stronger - just to avoid confusion which method to use:

"Use this method only when you explicitly need (...) Otherwise always prefer 'rule()' method".

Reply:
Done